packages/smooth_app/lib/pages/prices/prices_stats_page.dart

```diff
@@ -0,0 +1,661 @@
+                                  '${_getSafeInt('price_source_other_count')} | ${_getSafeInt('proof_source_other_count')}',
+                              label: 'Other',
+                            ),
+                          ),
+                        ],
+                      ),
+                      const SizedBox(height: 24),
+                      Center(
+                        child: Text(
+                          'Last updated on $lastUpdated',
+                          style: const TextStyle(
+                            color: Color(0xFFD3D3D3),
+                            fontSize: 12,
+                          ),
+                        ),
+                      ),
+                      const SizedBox(height: 16),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+    );
+  }
+}
+
+class CategoryHeader extends StatelessWidget {
+  const CategoryHeader({
+    super.key,
+    required this.icon,
+    required this.title,
+  });
+
+  final IconData icon;
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: <Widget>[
+        Icon(
+          icon,
+          color: Colors.white,
+          size: 22,
+        ),
+        const SizedBox(width: 8),
+        Text(
+          title,
+          style: const TextStyle(
+            fontSize: 20,
+            fontWeight: FontWeight.bold,
+            color: Colors.white,
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class StatsCard extends StatelessWidget {
+  const StatsCard({
+    super.key,
+    required this.number,
+    required this.label,
+    this.hasArrow = false,
+    this.url,
+  });
+
+  final String number;
+  final String label;
+  final bool hasArrow;
+  final String? url;
+
+  @override
+  Widget build(BuildContext context) {
+    final Widget card = Container(
+      padding: const EdgeInsets.all(16.0),
+      decoration: BoxDecoration(
+        color: const Color(0xFF303030),
+        borderRadius: BorderRadius.circular(6),
+      ),
+      child: Row(
+        children: <Widget>[
+          Expanded(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisSize: MainAxisSize.min,
+              children: <Widget>[
+                Text(
+                  number,
+                  style: const TextStyle(
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold,
+                    color: Colors.white,
+                  ),
+                ),
+                Text(
+                  label,
+                  style: const TextStyle(
+                    fontSize: 13,
+                    color: Color(0xFFBDBDBD),
+                  ),
+                ),
+              ],
+            ),
+          ),
+          if (hasArrow)
+            const Icon(
+              Icons.arrow_forward,
+              color: Colors.white,
+              size: 20,
```

Comment:
I have zero interest in UI, so please just keep the default sizes and colors. That way things will look good in dark/light modes.